etcd-configuration.adoc

```diff
@@ -4,19 +4,22 @@
 == etcd
 etcd is an open-source distributed, consistent key value store that provides a
 reliable way to store and access the data for shared configuration and service
-discovery across a cluster. Tendrl is using etcd to store the yaml files, other
-configuration files and its data.
+discovery across a cluster. Tendrl uses etcd as its central store to persist
+all the sds information, configuration definitions and tendrl management data.
 
 == Installation
-etcd can be installed in any server. However for tendrl application, it can be
-installed in the server where tendrl-api is installed. The package etcd can be
-installed by running "yum install etcd"
+The etcd package are available in epel. The epel repo can be installed using
```

Comment:
Yes, got it. The package is included in the CentOS Extras repository, enabled by default.